storm-core/src/jvm/org/apache/storm/command/KillTopology.java

```diff
@@ -31,6 +32,9 @@ public static void main(String[] args) throws Exception {
 
         @SuppressWarnings("unchecked")
         final List<String> names = (List<String>) cl.get("TOPO");
+        for (String name : names) {
```

Comment:
This should be treated differently because we have `continueOnError` option here.

Reply:
Changed the behavior that removing invalid name from the list and throw exception if no valid name is present.
I think this should has nothing to do with `continueOnError ` since the topology name is invalid and it makes no sense to proceed.